Core/AppRuntime/Source/AppRuntimeChakra.cpp

```diff
@@ -46,7 +46,7 @@ namespace Babylon
             &dispatchFunction));
         ThrowIfFailed(JsProjectWinRTNamespace(L"Windows"));
 
-#ifdef _DEBUG
+#if defined(_DEBUG) && !(defined(_M_ARM64) || defined(_M_ARM))
```

Comment:
I think it would be better to output a warning if it fails instead of throw. Does that work for you?

Reply:
Switched to no longer throw on failure but instead print a debug warning